changelogs/current.yaml

```diff
@@ -127,6 +127,10 @@ minor_behavior_changes:
 - area: router
   change: |
     Enable environment_variable in router direct response.
+- area: ext_authz
+  change: |
+    Ext_authz supports adding ``x-envoy-auth-failure-mode-allowed`` header optionally when ``failure_mode_allow`` is enable,
+    and runtime flag ``envoy.reloadable_features.http_ext_auth_failure_mode_allow_header_add`` is removed.
```

Comment:
This should go into the removed runtime section.

Reply:
done, PTAL again, thanks.